dump-format.mediawiki

```diff
@@ -0,0 +1,66 @@
+<pre>
+  NEP: TBD
+  Title: Unified Neo chain dump format
+  Author: Ekaterina Pavlova <ekt@nspcc.ru>, Roman Khimov <roman@nspcc.ru>
+  Type: Informational
+  Status: Draft
+  Created: 2025-02-14
+</pre>
+
+==Abstract==
+
+This proposal standardizes a single Neo chain dump format that unifies the “full” and “incremental”
+approaches into one consistent structure. It enables different Neo node implementations to remain
```

Comment:
```suggestion
approaches into one consistent structure. It enforces different Neo node implementations to remain
```

Reply:
`enables` seems to be more correct to me